test/__snapshots__/index.spec.ts.snap

```diff
@@ -552,32 +552,210 @@ exports[`eslintComments rules eslint-comments/no-use fails on an invalid fixture
 [
   {
     "column": 0,
-    "endColumn": 196,
+    "endColumn": 43,
     "endLine": 1,
     "line": 1,
     "message": "Unexpected ESLint directive comment.",
     "nodeType": null,
     "ruleId": "eslint-comments/no-use",
     "severity": 2,
   },
+  {
+    "column": 0,
+    "endColumn": 27,
+    "endLine": 3,
+    "line": 3,
+    "message": "Unexpected ESLint directive comment.",
+    "nodeType": null,
+    "ruleId": "eslint-comments/no-use",
+    "severity": 2,
+  },
+  {
+    "column": 0,
+    "endColumn": 25,
+    "endLine": 4,
+    "line": 4,
+    "message": "Unexpected ESLint directive comment.",
+    "nodeType": null,
+    "ruleId": "eslint-comments/no-use",
+    "severity": 2,
+  },
+  {
+    "column": 0,
+    "endColumn": 25,
+    "endLine": 5,
+    "line": 5,
+    "message": "Unexpected ESLint directive comment.",
+    "nodeType": null,
+    "ruleId": "eslint-comments/no-use",
+    "severity": 2,
+  },
   {
     "column": 1,
-    "endColumn": 196,
-    "endLine": 1,
-    "line": 1,
-    "message": "This line has a comment length of 195. Maximum allowed is 80.",
-    "messageId": "maxComment",
-    "nodeType": "Program",
-    "ruleId": "max-len",
+    "endColumn": 25,
+    "endLine": 5,
+    "line": 5,
+    "message": "Definition for rule 'foo' was not found.",
+    "nodeType": null,
+    "ruleId": "foo",
+    "severity": 2,
+  },
+  {
+    "column": 0,
+    "endColumn": 24,
+    "endLine": 6,
+    "line": 6,
+    "message": "Unexpected ESLint directive comment.",
+    "nodeType": null,
+    "ruleId": "eslint-comments/no-use",
     "severity": 2,
   },
   {
     "column": 1,
-    "fatal": true,
-    "line": 1,
-    "message": "Failed to parse JSON from ' "no-undef": off / / eslint-env browser / / eslint-disable foo / / eslint-enable bar / // eslint-disable-line // eslint-disable-next-line / exported foo / / global $ / / globals a, b, c': Unexpected token o in JSON at position 14",
```

Comment:
to me it looks like this test never worked correctly, it's not supposed to say "Unexpected token". I updated the fixture, see my comment in it below